json_object.c

```diff
@@ -167,8 +169,11 @@ static int json_escape_str(struct printbuf *pb, const char *str, int len, int fl
 extern struct json_object* json_object_get(struct json_object *jso)
 {
 	if (jso)
+	{
 		jso->_ref_count++;
-	return jso;
+		return jso;
+	}
+	return NULL;
```

Comment:
I agree with Keith here.  That's really a mostly pointless nano-optimization.  With optimization completely turned off, the single extra memory access this saves will be totally swamped by everything else, and for a case that will rarely be hit.  When optimization is turned on, this change makes no difference at all (jso is both passed and returned in a register).
It's not enough to preclude me from merging the pull request, but please don't bother spending any time on changes like this in the future, unless they're part of a more comprehensive effort to speed things up, and you have actual measurements to back it up.